src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest.cs

```diff
@@ -1922,6 +1922,10 @@ public void BuildWithNativeLibraries ([Values (true, false)] bool isRelease)
 							Assert.IsNotNull (data, "libMonoPosixHelper.so for x86 should exist in the apk.");
 							data = ZipHelper.ReadFileFromZip (zipFile, "lib/armeabi-v7a/libMonoPosixHelper.so");
 							Assert.IsNotNull (data, "libMonoPosixHelper.so for armeabi-v7a should exist in the apk.");
+							data = ZipHelper.ReadFileFromZip (zipFile, "lib/x86/llibmono-native.so");
```

Comment:
I think there's a typo here; is it *really* `llibmono-native.so` (*two* leading `l`s)?